"ci / test-mac (1.14, macos-10.15, 10.3)" and "ci / test-mac (1.13, macos-10.15, 10.3)" CI builds are failing

Error is

```
ld: unsupported tapi file type '!tapi-tbd' in YAML file '/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/lib/libSystem.tbd' for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: Process completed with exit code 2.
```

This has been discussed offline, filing to track.

I'll remove these builds from the required presubmit checks for now so that PRs are unblocked.

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -16,7 +16,7 @@ jobs:
         working-directory: ${{ env.WORKING_DIR }}
     strategy:
       matrix:
-        go: ['1.13', '1.14']
+        go: ['1.14', '1.15']
         os: ['macos-10.15']
         xcode-version: ['12.0', '11.7', '11.6', '11.5', '11.4.1', '11.3.1', '11.2.1', '11.1', '11.0']
     steps:
@@ -70,7 +70,7 @@ jobs:
         working-directory: ${{ env.WORKING_DIR }} 
     strategy:
       matrix:
-        go: ['1.13', '1.14']
+        go: ['1.14', '1.15']
         os: ['ubuntu-20.04', 'ubuntu-18.04', 'ubuntu-16.04'] 
     steps:
       - name: Update go version
@@ -114,7 +114,7 @@ jobs:
     runs-on: windows-2019
     strategy:
       matrix:
-        go: ['1.13', '1.14']
+        go: ['1.14', '1.15']
     steps:
       - name: Update go version
         uses: actions/setup-go@v2

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -18,7 +18,7 @@ jobs:
       matrix:
         go: ['1.13', '1.14']
         os: ['macos-10.15']
-        xcode-version: ['11.6', '11.5', '11.4.1', '11.3.1', '11.2.1', '11.1', '11.0', '10.3']
+        xcode-version: ['12.0', '11.7', '11.6', '11.5', '11.4.1', '11.3.1', '11.2.1', '11.1', '11.0']
     steps:
       - name: Update go version
         uses: actions/setup-go@v2

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,19 +13,19 @@ matrix:
       go: master
     - os: osx
       osx_image: xcode10.3
-      go: 1.13.x
+      go: 1.14.x
     - os: osx
       osx_image: xcode10.3
-      go: 1.14.x
+      go: 1.15.x
     - os: osx
       osx_image: xcode11.3
       go: master
     - os: osx
       osx_image: xcode11.3
-      go: 1.13.x
+      go: 1.14.x
     - os: osx
       osx_image: xcode11.3
-      go: 1.14.x
+      go: 1.15.x
     - os: osx
       osx_image: xcode11.6
       go: master

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,25 +12,25 @@ matrix:
       osx_image: xcode10.3
       go: master
     - os: osx
-      osx_image: xcode11
-      go: master
+      osx_image: xcode10.3
+      go: 1.13.x
     - os: osx
-      osx_image: xcode11.1
-      go: master
-    - os: osx
-      osx_image: xcode11.2
-      go: master
+      osx_image: xcode10.3
+      go: 1.14.x
     - os: osx
       osx_image: xcode11.3
       go: master
     - os: osx
-      osx_image: xcode11.4
-      go: master
+      osx_image: xcode11.3
+      go: 1.13.x
     - os: osx
-      osx_image: xcode11.5
-      go: master
+      osx_image: xcode11.3
+      go: 1.14.x
     - os: osx
       osx_image: xcode11.6
+      go: master
+    - os: osx
+      osx_image: xcode12
       go: master
 
 addons:

